corehq/apps/users/migrations/0019_editreportspermissions.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations
+
+from dimagi.utils.couch.database import iter_docs
+
+from corehq.apps.users.models import UserRole
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _migrate_edit_reports_permissions(apps, schema_editor):
+    roles = UserRole.view(
+        'users/roles_by_domain',
+        include_docs=False,
+        reduce=False
+    ).all()
+    for role_doc in iter_docs(UserRole.get_db(), [r['id'] for r in roles]):
+        role = UserRole.wrap(role_doc)
+
+        changed = False
+        if role.permissions.edit_data:
+            role.permissions.edit_reports = True
+            changed = True
+        elif role.permissions.edit_reports:
```

Comment:
proteusvacuum commented "I'm not sure I follow why this is being set to False here."

I borrowed this logic from the web apps permission change, which was added after the initial change:
https://github.com/dimagi/commcare-hq/blame/master/corehq/apps/users/migrations/0016_webappspermissions.py

It makes sure that edit_reports won't be true if edit_data is false.